src/sentry/auth/staff.py

```diff
@@ -179,24 +180,7 @@ def get_session_data(self, current_datetime=None):
             current_datetime = django_timezone.now()
 
         try:
-            data["idl"] = datetime.fromtimestamp(float(data["idl"]), timezone.utc)
-        except (TypeError, ValueError):
-            logger.warning(
-                "staff.invalid-idle-expiration",
-                extra={"ip_address": request.META["REMOTE_ADDR"], "user_id": request.user.id},
-                exc_info=True,
-            )
-            return
-
-        if data["idl"] < current_datetime:
-            logger.info(
-                "staff.session-expired",
-                extra={"ip_address": request.META["REMOTE_ADDR"], "user_id": request.user.id},
-            )
-            return
-
-        try:
-            data["exp"] = datetime.fromtimestamp(float(data["exp"]), timezone.utc)
+            expires_date = _seconds_to_timestamp(data["exp"])
```

Comment:
do we have to convert this every time? could we store it in the correct format in the first place 🤔

Reply:
I can try storing the datetime format itself

Reply:
ok jk due to serializing concerns it must be a timestamp